x-pack/plugins/apm/public/components/shared/KueryBar/Typeahead/Suggestion.js

```diff
@@ -17,31 +17,32 @@ import {
   unit
 } from '../../../../style/variables';
 import { tint } from 'polished';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
 
 function getIconColor(type) {
   switch (type) {
     case 'field':
       return colors.apmOrange;
     case 'value':
-      return colors.teal;
+      return theme.euiColorSecondary;
     case 'operator':
       return colors.apmBlue;
```

Comment:
Why not these?

Reply:
Sure, I can replace with the `euiColorVis` palette 👍